nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -55,6 +55,20 @@ let
     };
   };
 
+  mandatoryApps = [
+    "dav"
+    "files"
+    "federatedfilesharing"
+    "oauth2"
+    "profile"
+    "provisioning_api"
+    "settings"
+    "theming"
+    "twofactor_backupcodes"
+    "viewer"
+    "workflowengine"
+  ];
```

Comment:
This list should be read from the package: https://github.com/nextcloud/server/blob/2606f7b7dde9b3f252cc7d97fc7b285640e2c659/core/shipped.json#L101-L115

Reply:
ah nice! did not know that exists :tada: